hadoop-common-project/hadoop-kms/src/main/conf/kms-log4j.properties

```diff
@@ -34,7 +34,6 @@ log4j.additivity.kms-audit=false
 
 log4j.rootLogger=INFO, kms
 log4j.logger.org.apache.hadoop=INFO
-log4j.logger.com.sun.jersey.server.wadl.generators.WadlGeneratorJAXBGrammarGenerator=OFF
```

Comment:
This is related to HADOOP-15686. If I get HADOOP-15686 in, we'll have to update here.